README.md

```diff
@@ -1,11 +1,180 @@
 # CoreFx-TestData
 
-This repository contains test assets that are binary file or too large to be checked into corefx directly. Packages are produced and uploaded to the dotnet blob feed (https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json).
+This repository contains test assets that are binary file or too large to be checked into corefx directly. Packages are produced and uploaded to the [dotnet blob feed](https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json).
 
 Uploading the packages is currently a manual step. Community members can submit PRs but the deployment needs to be done by a .NET team member as described below.
 
 ## Workflow
-1. Make changes in the respective project and increments its version (usually just the patch fragment)
-2. Run `dotnet pack` on the project and find the produced nuget package in the project's bin folder.
-3. Use this build definition to upload the package to the dotnet blob feed: https://github.com/dotnet/core-eng/tree/master/Documentation/Tools/dotnet-core-push-oneoff-package. Make sure to clear the myget feed in the queue time variable section.
-4. Consume the updated package in corefx by incrementing the `PackageReference` version attribute: https://github.com/dotnet/corefx/blob/8d79b6117e9d584eebb8b6933bba83dd514010ca/external/test-runtime/XUnit.Runtime.depproj#L47-L54
+
+For the examples provided in the steps below, we are going to make the following assumptions:
+
+- We cloned the `dotnet/corefx-testdata` GitHub project in `D:\corefx-testdata`.
+- We cloned the `dotnet/corefx` GitHub project in `D:\corefx`.
+- We are working on adding a new unit test for the GZip feature from the `System.IO.Compression` namespace, and the test depends on a file called `example.gz`.
+
+#### 1. Modify the project files and increment the version number.
+
+We decided to save our `example.gz` file in `corefx-testdata\System.IO.Compression.TestData\GZipTestData\example.gz`.
+
+Now we edit the `corefx-testdata\System.IO.Compression.TestData\System.IO.Compression.TestData.csproj` file and bump the patch fragment of the version number.
+
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <PackageVersion>1.0.10</PackageVersion>
+    </PropertyGroup>
+</Project>
+```
+
+*Note: if the version was 1.0.9, bump it to 1.0.10, not 1.1.0.*
```

Comment:
I would remove this. Above you already mentioned that the part of the version to bump is the patch version, which is the number after the second dot. If anything, I would just add a parenthesis note after mentioning the word patch above specifying this is the third part of the version.

Reply:
Do you mean remove the example?

Reply:
nope, just remove the *Note